lib/tasks/populate_previous_entry.rake

```diff
@@ -0,0 +1,15 @@
+namespace :registers_frontend do
+  desc "populate previous entries"
+  task populate_previous_entry: :environment do
+    Spina::Register.find_each do |register|
+      entries_by_key =  Entry.where(spina_register_id: register.id).group_by(&:key)
+      entries_with_history = entries_by_key.select { |_key, entries| entries.length > 1 }
+      entries_with_history.each do |key, entries|
+        entries.each_cons(2) { |current, previous|
+          puts "register #{register.name} setting key #{key} #{current.entry_number} previous entry to #{previous.entry_number}"
+          Entry.update(current.id, previous_entry_number: previous.entry_number)
+        }
```

Comment:
It's considered to be good ruby style to use `do/end` for multiline blocks and `{}` for oneliners.

Reply:
@gidsg  Github lost track of this comment but it is still relevant.